scout/server/links.py

```diff
@@ -172,6 +172,18 @@ def omim(omim_id):
     return link.format(omim_id)
 
 
+def disease_link(disease_id: str) -> str:
+    """Returns a link for disease depending on the source of the term"""
+    source, disease_nr = disease_id.split(":")
+    link = None
+    if source == "OMIM":
+        link = f"https://www.omim.org/entry/{disease_nr}"
+    elif source == "ORPHA":
+        link = f"https://www.orpha.net/consor/cgi-bin/OC_Exp.php?lng=en&Expert={disease_nr}"
```

Comment:
Doesn't look like it leads to the intended page

Reply:
I swear this morning it just went to a search page, with no results under. Weird! Never mind!